openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -259,3 +242,34 @@ def get_credit_provider_display_names(course_key):
         cache.set(cache_key, provider_names, credit_config.cache_ttl)
 
     return provider_names
+
+
+def make_providers_strings(providers):
+    """Get the list of course providers and make them comma seperated string.
+
+    Arguments:
+        providers : List containing the providers names
+
+    Returns:
+        strings containing providers names in readable way .
+    """
+    providers_string = None
+    if providers:
+        if len(providers) > 1:
+            if len(providers) > 2:
+                # Translators: The join of three or more university names. The first of these formatting strings
+                # represents a comma-separated list of names (e.g., MIT, Harvard, Dartmouth).
+                providers_string = _("{first_providers}, and {last_provider}").format(
```

Comment:
No need to remove.
https://github.com/edx/edx-platform/commit/1d1baa50098c1108a4ae7e0c2fc345fbc8dd8a3c#diff-d8e9b05acd3b7dba84709ae01b99053cR858